ironfish-cli/src/commands/miners/mined.ts

```diff
@@ -46,6 +46,14 @@ export class MinedCommand extends IronfishCommand {
 
     this.log('Scanning for mined blocks...')
 
+    //Truncate fractions from parameters
+    args.stop = MathUtils.floor(args.stop, 0)
```

Comment:
This is a bit of a typescript thing, but look below at lines 58. It's typing these differently as `number | null` which means the underlying value is nullable because above the required is false. If they were typed properly from our CLI library then it would cause an error here.

If you look at MathUtils.floor it doesn't take `number | null` only `number` so were actually hiding a bug here.  When typescript libraries can't type things properly, they'll use any which means the type system is disabled. Consider this code, I'd say it has unexpected output.

```typescript
function addOne(value: number): number {
    return value + 1
}

const a: any = 1
const b: any = null

console.log(addOne(a))
console.log(addOne(b))
```

Reply:
Sure